docroot/modules/contrib/ova/ova.module

```diff
@@ -0,0 +1,138 @@
+<?php
+
+/**
+ * @file
+ * Open Video Annotation using videojs HTML5 player.
+ */
+
+/**
+ * Implements hook_theme().
+ */
+function ova_theme() {
+  return array(
+    'ovaAudio' => array(
+      'variables' => array(
+        'items' => NULL,
+        'player_extension' => NULL,
+        'player_attributes' => NULL,
+        'player_image_path' => NULL,
+        'entity' => NULL,
+      ),
+      'file' => 'ova.inc',
+    ),
+    'ovaVideo' => array(
+      'variables' => array(
+        'items' => NULL,
+        'player_extension' => NULL,
+        'player_attributes' => NULL,
+        'entity' => NULL,
+        'player_image_path' => NULL,
+      ),
+      'file' => 'ova.inc',
+    ),
+  );
+}
+
+/**
+ * Implements hook_preprocess_HOOK().
+ */
+function ova_preprocess_ova(&$variables) {
+
+  $node = \Drupal::routeMatch()->getParameter('node');
+  $variables['get_nid'] = $node->id();
+}
+
+/**
+ * Implements hook_annotation_store_search_endpoint_output_alter(&$entity).
+ */
+function ova_annotation_store_search_endpoint_output_alter(&$entity) {
```

Comment:
Doesn't this mean the module is dependent on annotation_store module? If this is built generic for use by others, i would remove this part out of the module & implement this hook in a custom module.